css/edit-block.css

```diff
@@ -0,0 +1,114 @@
+body {
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+}
+
+input[type="radio"]:checked + div {
+	border-color: rgb(0, 124, 186);
+}
+
+.side {
+	width: 280px;
+}
+
+.side input[type="checkbox"] {
+	display: block;
+	margin-top: 0.5rem;
+}
+
+.field-resize {
+	cursor: col-resize;
+}
+
+.block-title-field {
+	font-size: 2rem;
+}
+
+.repeater-insert-row button {
+	display: none;
+}
+
+.repeater-insert-row:hover button {
+	display: flex;
+}
+
+.repeater-row__move:after {
+	content: "";
+	display: block;
+	position: absolute;
+	height: 300%;
+	width: 2.5rem;
+	top: -.5rem;
+	right: -.5rem;
+	/* border:1px solid red; */
+}
+
+.repeater-row__delete-duplicate,
+.repeater-row__move {
+	display: none;
+}
+
+.repeater-row:hover .repeater-row__delete-duplicate,
+.repeater-row:hover .repeater-row__move {
```

Comment:
Some of these rules aren't for actual elements now, they'll be added in upcoming PRs.